src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1511,7 +1511,7 @@ def index(self) -> Union["pd.Index", native_pd.MultiIndex]:
         if self.is_multiindex():
             return self._modin_frame.index_columns_pandas_index
         else:
-            return pd.Index(self)
+            return pd.Index(self, tupleize_cols=False)
```

Comment:
why do we need tupleize_cols=False here?

Reply:
In the case where the index is lists, we do not want to create a multi-index